app/services/push_service.rb

```diff
@@ -1,12 +1,17 @@
 class PushService
-  def send_update_store(channel, app_name, data)
-    channel = Channel.add_app_name_suffix(channel, app_name)
-    SocketioSendJob.perform_later(channel, "update_store", data.to_json, true)
+  def send_update_store(channels, app_name, data)
+    # TODO remove add_app_name_suffix, decide correct channels further up
+    channels = [channels].flatten.uniq
+    channels = Channel.add_app_name_suffix(channels, app_name)
+    if channels.any?
+      SocketioSendJob.perform_later(channels, "update_store", data.to_json, true)
+    end
```

Comment:
@abhinav098 just wanted to note a subtle change here. We will allow `send_update_store` to receive `channel` argument as either a string (single channel) or an array of channels. It doesn't matter which it receives as it will flatten and uniq the channels before queuing the job. Doing it here means we won't need to worry about it everywhere else we call the code.